defs.bzl

```diff
@@ -38,7 +38,7 @@ rollup_bundle = _rollup_bundle
 npm_package = _npm_package
 # ANY RULES ADDED HERE SHOULD BE DOCUMENTED, run yarn skydoc to verify
 
-def node_modules_filegroup(packages, **kwargs):
+def node_modules_filegroup(packages, patterns=[], **kwargs):
```

Comment:
where would this be used? like for adding `history-server`? I think we should move away from this rule for when `preserve-symlinks` is on by default

Reply:
Agreed on moving away from it. I needed this for the protractor rule:
```
node_modules_filegroup(
    name = "node_modules",
    packages = [...]
    patterns = [
        "node_modules/protractor/**",
    ],
)
```

Reply:
I think its useful to get in anyway even if angular doesn't use it in the long run

Reply:
okay makes sense